app/uk/gov/hmrc/eoricommoncomponent/frontend/views/check_your_details_register.scala.html

```diff
@@ -451,11 +431,11 @@
         ),
         summaryListRowNoChangeOption(
             key = messages("cds.form.gb-vat-date"),
-            value = Some(Html(subscription.vatControlListResponse.map(vat => formatDate(DateConverter.toLocalDate(vat.dateOfReg.get).get)).getOrElse(messages("cds.not-entered.label")))),
+            value = Some(Html(subscription.vatControlListResponse.map(vat => formatDate(LocalDate.parse(vat.dateOfReg.get))).getOrElse(messages("cds.not-entered.label")))),
```

Comment:
I've changed it now to throw DataUnavailableException on the getOrElse because we need the data to exist in order to parse the date correctly, but at least this is safer now 👍